src/API/Management/Clients.php

```diff
@@ -1,101 +1,149 @@
 <?php
-
+/**
+ * Clients endpoints for the Management API.
+ *
+ * @package Auth0\SDK\API\Management
+ */
 namespace Auth0\SDK\API\Management;
 
-
-use Auth0\SDK\API\Header\ContentType;
-
-class Clients extends GenericResource 
+/**
+ * Class Clients.
+ * Handles requests to the Clients endpoint of the v2 Management API.
+ *
+ * @package Auth0\SDK\API\Management
+ */
+class Clients extends GenericResource
 {
+
     /**
-     * @param null|string|array $fields
-     * @param null|string|array $include_fields
+     * Get all Clients by page.
+     *
+     * @param null|string|array $fields         - Fields to include or exclude from the result.
+     * @param null|boolean      $include_fields - True to include $fields, false to exclude $fields.
+     * @param integer           $page           - Page number to get, zero-based.
+     * @param null|integer      $per_page       - Number of results to get, null to return the default number.
+     * @param array             $add_params     - Additional API parameters; see docs link below for supported.
+     *
      * @return mixed
+     *
+     * @throws \Exception
+     *
+     * @link https://auth0.com/docs/api/management/v2#!/Clients/get_clients
      */
-    public function getAll($fields = null, $include_fields = null) 
+    public function getAll($fields = null, $include_fields = null, $page = 0, $per_page = null, $add_params = [])
     {
-        $request = $this->apiClient->get()
-            ->clients();
+        // Set additional parameters first so they are over-written by function parameters.
```

Comment:
this is fine, but would be nice to get it clearer on the docblock for this add_params param. e.g. "the values specified here are overwritten by the previous params" or something like that